README.md

```diff
@@ -13,7 +13,8 @@
 
 ## Introduction 🎉
 
-This library provides bit security estimators and asymptotic complexity estimators for cryptographic problems. Some of the implemented estimators are:
+This library provides bit complexity estimators for cryptographic problems and bit security estimators for some cryptographic schemes. 
+Currently, the implemented estimators are:
```

Comment:
This library provides bit complexity estimators for cryptographic problems, as well as some cryptographic schemes. 
Currently, the implemented estimators are: